### ISSUE TITLE
Filter implementation?

### ISSUE THREAD

**Issue by DatanoiseTV** [raybellis/PicoSynth#2]
Are you planning on adding a filter?

You could probably reuse some code generated by Vult DSP. There is a "ladder-type" low-pass filter
and a SVF filter implementation I could share.

**Comment by DatanoiseTV** [comment]
Vult is a transcompiler which can compile to C++ along other languages.

Here is the filter code (in Vult DSP language):
https://github.com/vult-dsp/vult/tree/master/examples/filters

**Comment by DatanoiseTV** [comment]
[ladder_filt_vult_gen.zip](https://github.com/raybellis/PicoSynth/files/13841332/ladder_filt_vult_gen.zip)

**Comment by DatanoiseTV** [comment]
[filter_impl_vult_c_fixedpoint.zip](https://github.com/raybellis/PicoSynth/files/13841336/filter_impl_vult_c_fixedpoint.zip)

**Comment by raybellis** [comment]
Yes, a filter is the highest priority for the next feature.  There's actually a `filter` branch where I started work on a state-variable-filter, but I was having trouble finding a decent implementation that would be stable at higher cutoff frequencies.
